packages/client/src/components/canary-executor/ScopeItem.tsx

```diff
@@ -76,9 +113,10 @@ export default class ScopeItem extends React.Component<ScopeProps> {
         <InlineTextGroup
           id={scopeType + '-location'}
           label="Location"
+          placeHolderText="The location of your deployed service."
```

Comment:
should these be flipped?
Where the description is the subtext, so users always see it,
and then the placeholder text is the example?

Reply:
Let's talk about this later, and we can tweak these and talk about what belongs where in regards to the placeholders, subtext and popovers (tooltips).